### PR TITLE
Fix recommended `download-logs` command in `bbs2gh migrate-repo` and `ado2gh migrate-repo` to use correct arguments

### PR DESCRIPTION
**Pull request by timrogers**
When you run the `migrate-repo` command, as part of the output, we show you how to download your migration logs with the `download-logs` command, providing copy and paste commands.

Currently, those copy and paste commands don't work because they use incorrect arguments. This fixes them to use the correct arguments.

- [x] Did you write/update appropriate tests
- [x] Release notes updated (if appropriate)
- [x] Appropriate logging output
- [ ] Issue linked
- [ ] Docs updated (or issue created)
- [ ] New package licenses are added to `ThirdPartyNotices.txt` (if applicable)

### CODE REVIEW

**Comment by begonaguereca** [comment]
Is there any corresponding documentation that needs to be updated?

**Comment by timrogers** [comment]
I've asked @begonaguereca to take a look at this.

For testing locally, you can run the `ado2gh` and `bbs2gh` CLIs like this, replacing `$COMMAND` with a specific command (e.g. `migrate-repo`):

```bash
dotnet run --project src/bbs2gh/bbs2gh.csproj -- $COMMAND
dotnet run --project src/ado2gh/ado2gh.csproj -- $COMMAND
```

There are two ways you can test this change:

1. Call the `download-logs` command using the arguments before and after this change. You'll see that the current CLI options are not correct!
2. Run migrations using `migrate-repo` and copy and paste a real command outputted by the CLI, for a true end to end test

**Comment by timrogers** [comment]
A quick explanation that shows that the current args are wrong:

You can see here that the args are wrong:

* These classes define the `download-logs` command for bbs2gh and ado2gh: https://github.com/github/gh-gei/blob/main/src/bbs2gh/Commands/DownloadLogs/DownloadLogsCommand.cs, https://github.com/github/gh-gei/blob/main/src/ado2gh/Commands/DownloadLogs/DownloadLogsCommand.cs
- Both depend on `DownloadLogsCommandBase`: https://github.com/github/gh-gei/blob/main/src/Octoshift/Commands/DownloadLogs/DownloadLogsCommandBase.cs
- `DownloadLogsCommandBase` defines `GitHubOrg` and `GitHubRepo` args

**Comment by begonaguereca** [comment]
Looks like there is no description for: 

`github-org` and `github-repo`

```
Options:
  --archive-url <archive-url>                                          URL used to download Bitbucket Server migration archive. Only needed if you want to manually 
                                                                       retrieve the archive from BBS instead of letting this CLI do that for you.
  --github-org <github-org>
  --github-repo <github-repo>
  --github-pat <github-pat>                                            The GitHub personal access token to be used for the migration. If not set will be read from 
                                                                       GH_PAT environment variable.
  --bbs-server-url <bbs-server-url> (REQUIRED)                         The full URL of the Bitbucket Server/Data Center to migrate from. E.g. 
                                                                       http://bitbucket.contoso.com:7990
  --bbs-project <bbs-project> (REQUIRED)                               The Bitbucket project to migrate.
  --bbs-repo <bbs-repo> (REQUIRED)                                     The Bitbucket repository to migrate.
  --bbs-username <bbs-username>                                        The Bitbucket username of a user with site admin privileges. If not set will be read from 
                                                                       BBS_USERNAME environment variable.
```

**Comment by begonaguereca** [comment]
Sorry for being a newb here, I'm not understanding where `--download-logs` is being used. 

I tried running the following command: 
`dotnet run --project src/bbs2gh/bbs2gh.csproj -- migrate-repo --bbs-server-url https://test-bbs-o.githubapp.com/ --bbs-project IM --bbs-repo test-commit-id-blank-review --download-logs`

also tried running the help command and don't see it as an option there

**Comment by timrogers** [comment]
@begonaguereca No worries! `download-logs` is a command in itself, so you'd run something like this:

> dotnet run --project src/bbs2gh/bbs2gh.csproj -- download-logs --help

That will output the supported arguments for `download-logs`, which already demonstrates that the example commands we currently output when you run a migration are invalid.

**Comment by timrogers** [comment]
This PR doesn't actually make any changes to the `download-logs` command itself - it just fixes some places where the CLI outputs example `download-logs` commands which are wrong, because they use non-existent arguments.

**Comment by begonaguereca** [comment]
Also i think we need to [update](https://docs.github.com/en/migrations/using-github-enterprise-importer/completing-your-migration-with-github-enterprise-importer/accessing-your-migration-logs-for-github-enterprise-importer) these docs

**Comment by begonaguereca** [comment]
Main: 

```
Your branch is up to date with 'origin/main'.
begonaguereca@Begonas-MacBook-Pro gh-gei % dotnet run --project src/ado2gh/ado2gh.csproj -- download-logs --help
[2023-10-18 15:10:00] [WARNING] You are not running the ado2gh CLI as a gh extension. This is not recommended, please run: gh extension install github/gh-ado2gh
[2023-10-18 15:10:01] [WARNING] You are running an older version of the ado2gh CLI [v1.0.0.0]. The latest version is v1.2.1.
[2023-10-18 15:10:01] [WARNING] Please update by running: gh extension upgrade ado2gh
Description:
  Downloads migration logs for migrations.

Usage:
  ado2gh download-logs [options]

Options:
  --github-org <github-org> (REQUIRED)       GitHub organization to download logs from.
  --github-repo <github-repo> (REQUIRED)     Target repository to download latest log for.
  --github-api-url <github-api-url>          Target GitHub API URL if not targeting github.com (default: https://api.github.com).
  --github-pat <github-pat>                  Personal access token of the GitHub target. Overrides GH_PAT environment variable.
  --migration-log-file <migration-log-file>  Local file to write migration log to (default: migration-log-ORG-REPO-MIGRATION_ID.log).
  --overwrite                                Overwrite migration log file if it exists.
  --verbose                                  Display more information to the console.
  -?, -h, --help                             Show help and usage information
```

branch: `timrogers/download-logs-suggested-commands`

```
Your branch is up to date with 'origin/timrogers/download-logs-suggested-commands'
begonaguereca@Begonas-MacBook-Pro gh-gei % dotnet run --project src/ado2gh/ado2gh.csproj -- download-logs --help
[2023-10-18 15:10:30] [WARNING] You are not running the ado2gh CLI as a gh extension. This is not recommended, please run: gh extension install github/gh-ado2gh
[2023-10-18 15:10:30] [WARNING] You are running an older version of the ado2gh CLI [v1.0.0.0]. The latest version is v1.2.1.
[2023-10-18 15:10:30] [WARNING] Please update by running: gh extension upgrade ado2gh
Description:
  Downloads migration logs for migrations.

Usage:
  ado2gh download-logs [options]

Options:
  --github-org <github-org> (REQUIRED)       GitHub organization to download logs from.
  --github-repo <github-repo> (REQUIRED)     Target repository to download latest log for.
  --github-api-url <github-api-url>          Target GitHub API URL if not targeting github.com (default: https://api.github.com).
  --github-pat <github-pat>                  Personal access token of the GitHub target. Overrides GH_PAT environment variable.
  --migration-log-file <migration-log-file>  Local file to write migration log to (default: migration-log-ORG-REPO-MIGRATION_ID.log).
  --overwrite                                Overwrite migration log file if it exists.
  --verbose                                  Display more information to the console.
  -?, -h, --help                             Show help and usage information
```

They look the same to me, am I missing something?

**Comment by timrogers** [comment]
> Also i think we need to [update](https://docs.github.com/en/migrations/using-github-enterprise-importer/completing-your-migration-with-github-enterprise-importer/accessing-your-migration-logs-for-github-enterprise-importer) these docs

Great spot! Those docs are also wrong 😓 

> They look the same to me, am I missing something?

I'll set up some time to talk through this later ❤️

File: src/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs [hunk 1]
```diff
@@ -86,13 +86,13 @@ public async Task Handle(MigrateRepoCommandArgs args)
         {
             _log.LogError($"Migration Failed. Migration ID: {migrationId}");
             _warningsCountLogger.LogWarningsCount(warningsCount);
-            _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `gh {CliContext.RootCommand} download-logs --github-target-org {args.GithubOrg} --target-repo {args.GithubRepo}`");
```
Comment (begonaguereca) [inline src/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs:89]: Can we throw this into a variable? So that it only needs to be updated once?

**Comment by begonaguereca** [comment]
Ok for a successful migration it is working! 

`main`: ❌  

⛵ `dotnet run --project src/ado2gh/ado2gh.csproj -- migrate-repo --ado-org "valet-testing" --ado-team-project "build-playground" --ado-repo "forecast-regression-test" --github-org "import-testing" --github-repo "unicorn-main" --verbose`

```
[2023-10-19 08:28:49] [INFO] Migration completed (ID: RM_kgDaACRmNTg0YWY5NC1kZjZlLTQ5MzUtODc4MC04YzQxZGY1MzliMDM)! State: SUCCEEDED
[2023-10-19 08:28:49] [INFO] Migration log available at https://objects-origin.githubusercontent.com/octoshiftmigrationlogs/import-testing/unicorn-main-f584af94-df6e-4935-8780-8c41df539b03-logs.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=octoshiftstoragedev%2F20231019%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231019T152839Z&X-Amz-Expires=432000&X-Amz-Signature=96bafa2f80fd5533be0e764cbf41b9650084244fc90c92ff4b9b2f5086c1c5c9&X-Amz-SignedHeaders=host or by running `gh ado2gh download-logs --github-target-org import-testing --target-repo unicorn-main`
```

⛵ `gh ado2gh download-logs --github-target-org import-testing --target-repo unicorn-main`

```
[2023-10-19 08:29:33] [WARNING] You are running an older version of the ado2gh CLI [v1.2.0.0]. The latest version is v1.2.1.
[2023-10-19 08:29:33] [WARNING] Please update by running: gh extension upgrade ado2gh
Option '--github-org' is required.
Option '--github-repo' is required.
Unrecognized command or argument '--github-target-org'.
Unrecognized command or argument 'import-testing'.
Unrecognized command or argument '--target-repo'.
Unrecognized command or argument 'unicorn-main'.
```
-------------

`timrogers/download-logs-suggested-commands` ✅ 

🚗 `dotnet run --project src/ado2gh/ado2gh.csproj -- migrate-repo --ado-org "valet-testing" --ado-team-project "build-playground" --ado-repo "forecast-regression-test" --github-org "import-testing" --github-repo "unicorn-main" --verbose`

```
[2023-10-19 08:15:53] [INFO] Migration completed (ID: RM_kgDaACQ4YmU2ODA5ZS0wNjBkLTQ0NzAtYWEzOC04Mzc0MWRiOTVhOTI)! State: SUCCEEDED
[2023-10-19 08:15:53] [WARNING] 1 warning encountered during this migration
[2023-10-19 08:15:53] [INFO] Migration log available at https://objects-origin.githubusercontent.com/octoshiftmigrationlogs/import-testing/unicorn-8be6809e-060d-4470-aa38-83741db95a92-logs.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=octoshiftstoragedev%2F20231019%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231019T151543Z&X-Amz-Expires=432000&X-Amz-Signature=52365bf878754944d80ba7b07887c26e365601a66cec306d6b1c2057bb936f07&X-Amz-SignedHeaders=host or by running `gh ado2gh download-logs --github-org import-testing --github-repo unicorn`
```

Followed by:
🚗 ` gh ado2gh download-logs --github-org import-testing --github-repo unicorn`

```
[2023-10-19 08:16:22] [INFO] GITHUB ORG: import-testing
[2023-10-19 08:16:22] [INFO] GITHUB REPO: unicorn
[2023-10-19 08:16:22] [WARNING] Migration logs are only available for 24 hours after a migration finishes!
[2023-10-19 08:16:22] [INFO] Downloading migration logs...
[2023-10-19 08:16:23] [INFO] Downloading log for repository unicorn to migration-log-import-testing-unicorn-RM_kgDaACQ4YmU2ODA5ZS0wNjBkLTQ0NzAtYWEzOC04Mzc0MWRiOTVhOTI.log...
[2023-10-19 08:16:23] [INFO] Downloaded unicorn log to migration-log-import-testing-unicorn-RM_kgDaACQ4YmU2ODA5ZS0wNjBkLTQ0NzAtYWEzOC04Mzc0MWRiOTVhOTI.log.
```

**Comment by begonaguereca** [comment]
and works for a "failed migration" when the project doesn't exist! ✅

File: src/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs (hunk 1 above)
Comment (timrogers) [reply src/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs:89]: Done in a1e0897

**Review by begonaguereca — APPROVED** [review]
🚀 LGTM!

**Comment by begonaguereca** [comment]
@timrogers  sorry one last thing, what would you think about adding a new line to separate the hyperlink and the command ? 

The hyperlink is so long it makes I feel like the command gets lost in the mix: 

Current: 
```
Migration log available at https://objects-origin.githubusercontent.com/octoshiftmigrationlogs/import-testing/unicorn-8be6809e-060d-4470-aa38-83741db95a92-logs.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=octoshiftstoragedev%2F20231019%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231019T151543Z&X-Amz-Expires=432000&X-Amz-Signature=52365bf878754944d80ba7b07887c26e365601a66cec306d6b1c2057bb936f07&X-Amz-SignedHeaders=host or by running `gh ado2gh download-logs --github-org import-testing --github-repo unicorn`
```

Potential: 
```
Migration log: https://objects-origin.githubusercontent.com/octoshiftmigrationlogs/import-testing/unicorn-8be6809e-060d-4470-aa38-83741db95a92-logs.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=octoshiftstoragedev%2F20231019%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231019T151543Z&X-Amz-Expires=432000&X-Amz-Signature=52365bf878754944d80ba7b07887c26e365601a66cec306d6b1c2057bb936f07&X-Amz-SignedHeaders=host

or by running the following command: `gh ado2gh download-logs --github-org import-testing --github-repo unicorn`
```

**Pull request merged**